front/SettingsForm.tsx

```diff
@@ -98,6 +101,7 @@ const SettingsForm = () => {
             <input
               className="form-control form-control-lg"
               type="password"
+              data-cy = "settings-newPassword"
```

Comment:
Remove the space between 'data-cy' and '='. It should be written as 'data-cy="settings-newPassword"'